setup

```diff
@@ -1,4 +1,4 @@
-#! /bin/sh -
+#! /bin/bash -
```

Comment:
This is not valid on FreeBSD 😈 . Bash is not installed by default and even if it is, then it could be found under `/usr/local/bin/bash`.

Additionally, `$()` seems to be a POSIX syntax: http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_03

Reply:
Hmm, okay. I guess I was wrong then! My vim complains about it, unless I change the shebang line. I'll drop that commit, and re-pull.

Goat does look to be super useful, and I'm excited to start using it. :-)